components/ide/jetbrains/backend-plugin/build.gradle.kts

```diff
@@ -13,7 +13,7 @@ plugins {
     // Kotlin support - check the latest version at https://plugins.gradle.org/plugin/org.jetbrains.kotlin.jvm
     id("org.jetbrains.kotlin.jvm") version "1.7.0"
     // gradle-intellij-plugin - read more: https://github.com/JetBrains/gradle-intellij-plugin
-    id("org.jetbrains.intellij") version "1.8.0"
+    id("org.jetbrains.intellij") version "1.8.1"
```

Comment:
ℹ️ Not part of the required changes, but Gradle keeps telling us to update it whenever we build the plugin, so it's better to keep it up to date.